# usefullinks

IDR Chip-seq: https://hbctraining.github.io/Intro-to-ChIPseq/lessons/07_handling-replicates-idr.html

Remap 2020: http://remap.univ-amu.fr/

About transcriptome based rna-seq: 

https://cgatoxford.wordpress.com/2016/08/17/why-you-should-stop-using-featurecounts-htseq-or-cufflinks2-and-start-using-kallisto-salmon-or-sailfish/

Using binder with R:

https://github.com/alan-turing-institute/the-turing-way/blob/master/workshops/boost-research-reproducibility-binder/workshop-presentations/zero-to-binder-r.md

Recount3:
http://rna.recount.bio/

Snakemake checkpoints:
http://ivory.idyll.org/blog/2021-snakemake-checkpoints.html

conda env on hpc:
https://kb.iu.edu/d/axgp#conda-init

Reproducible toolkit:
https://rdmkit.elixir-europe.org/index.html

Interoperability platform:
https://elixir-europe.org/platforms/interoperability

workflow hub:
https://workflowhub.eu/

guide to reproducible research:
https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html

galaxy admin: https://github.com/galaxyproject/admin-training

fastq repo: ftp://ftp.sra.ebi.ac.uk/vol1/fastq/

kahoot: kahoot.it / kahoot.com

Elixir train the trainer: https://docs.google.com/document/d/1ZkwDJDii5nHFPDsiMto-Fwyw9W1od9IHDzJL_t-uuR8/edit#

ATAC-Seq video:
https://www.youtube.com/watch?v=VZFUu_cJxyI

CookieCutter:
https://medium.com/worldsensing-techblog/project-templates-and-cookiecutter-6d8f99a06374

Multi-omics workshop:
https://uppsala.instructure.com/courses/52162/pages/schedule

Authorships:
https://credit.niso.org/
http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html
https://publicationethics.org/authorship

Publication ethics:
https://www.councilscienceeditors.org/resource-library/editorial-policies/white-paper-on-publication-ethics/

EMBL open science:
https://www.embl.org/internal-information/wp-content/uploads/2021/12/open-science-implementation-guidelines.pdf

The Declaration on Research Assessment:
https://sfdora.org/

Multi-stages singularity by Francesco:
https://git.embl.de/tabaro/snakemake-pipelines/-/blob/main/singularity/Singularity.bedtools

singularity CI by Francesco:
https://git.embl.de/tabaro/snakemake-pipelines/-/blob/main/.gitlab-ci.yml

singularity CI by IT:
https://git.embl.de/grp-itservices/singularity-reg/-/blob/master/.gitlab-ci.yml

spatial-transcriptomics:
https://www.nature.com/articles/s41587-021-01006-2#Sec9
https://www.nature.com/articles/s41586-019-1049-y
https://biofam.github.io/MOFA2/MEFISTO.html
https://squidpy.readthedocs.io/en/stable/auto_tutorials/tutorial_seqfish.html
https://github.com/spacetx/starfish#starfish-scalable-pipelines-for-image-based-transcriptomics

Graphical access with x2go:
https://wiki.embl.de/cluster/Env#Graphical_Access

Workflowhub:
https://about.workflowhub.eu/

Workflows community initiative:
https://workflows.community/

Look at docker recipe: https://git.embl.de/kaspar/learnr-project

Awsome pipelines repository:
https://github.com/pditommaso/awesome-pipeline

Nextflow tutorial to start with:
https://sateeshperi.github.io/nextflow_varcal/nextflow/

Nextflow tutorials:
https://nf-co.re/docs/usage/nextflow

About managing a core:
https://www.researchcomputingteams.org/resources

JBrowse 2:
https://jbrowse.org/jb2/

Visual integration tool for exploration of spatial single cell experiments:
http://vitessce.io/

human body at single-cell resolution:
https://hubmapconsortium.org/

psupertime: supervised pseudotime analysis for time-series single-cell RNA-seq data
https://academic.oup.com/bioinformatics/article/38/Supplement_1/i290/6617492

About using nextflow:
https://www.reddit.com/r/bioinformatics/comments/sc8e0i/does_everyone_use_nextflow_thesedays/?utm_source=share&utm_medium=web2x&context=3

spacemake:
https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giac064/6646447?login=true

--------------------- Aug 2022

Neuroscience Cloud Analysis As a Service:
https://www.biorxiv.org/content/10.1101/2020.06.11.146746v2

Choose your statistical test:
https://www.statstest.com/

Collection of singularities:
https://depot.galaxyproject.org/singularity/

open ebench:
https://openebench.bsc.es/benchmarking

Workflow finder:
https://australianbiocommons.github.io/2_1_workflows.html

GitHub Actions Tutorial - Basic Concepts and CI/CD Pipeline with Docker:
https://www.youtube.com/watch?v=R8_veQiYBjI

Dockerhub best practices:
https://www.docker.com/blog/best-practices-for-using-docker-hub-for-ci-cd/

Publishing docker images with github actions:
https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

snakemake tutorial:
https://edcarp.github.io/2022-11-08_ed-dash_workflows-snakemake/

Nextflow tutorial:
https://edcarp.github.io/2023-01-24_ed-dash_workflows-nextflow/
